src/souschef/section.py

```diff
@@ -74,5 +74,4 @@ def value(self, items):
             self._parent()[self._name] = items
 
     def update(self, section: Mapping):
```

Comment:
indeed, you can add Sequence there as well then please?
```suggestion
    def update(self, section: Mapping | Sequence):
```
you might need to import it

Reply:
In fact it should be `Any` I suppose. See:

```
    @value.setter
    def value(self, items):
        if isinstance(items, abc.Mapping):

        elif isinstance(items, str):

        elif isinstance(items, abc.Sequence):

        else:
            self._parent()[self._name] = items
```

Reply:
yeah, indeed in this case should be any. The original idea of ``update`` was to be used just with a dict passed as parameter. It could handle more but the initial idea was it

Reply:
Done

Reply:
cool! Thanks for that! :)